Source/SwiftLintFramework/Reporters/HTMLReporter.swift

```diff
@@ -120,7 +120,7 @@ public struct HTMLReporter: Reporter {
     private static func generateSingleRow(for violation: StyleViolation, at index: Int) -> String {
         let severity: String = violation.severity.rawValue.capitalized
         let location = violation.location
-        let file: String = location.file ?? ""
+        let file: String = location.file?.escapedForXml() ?? ""
```

Comment:
we should probably use `"<nopath>"` here too for consistency with `CheckstyleReporter.swift`: `(location.file ?? "<nopath>").escapedForXml()`